declaration/AnimationManager.d.ts

```diff
@@ -18,25 +18,31 @@ export interface AnimationParam {
     input?: IInputType;
 }
 export declare class AnimationManager {
-    private options;
-    private itm;
-    private em;
-    private axm;
     private _raf;
     private _animateParam;
+    private options;
+    itm: InterruptManager;
+    em: EventManager;
+    axm: AxisManager;
     static getDuration(duration: number, min: number, max: number): number;
-    constructor(options: AxesOption, itm: InterruptManager, em: EventManager, axm: AxisManager);
+    constructor({options, itm, em, axm}: {
+        options: any;
+        itm: any;
+        em: any;
+        axm: any;
+    });
     getDuration(depaPos: Axis, destPos: Axis, wishDuration?: number): number;
-    private createAnimationParam(pos, duration, inputEvent?);
-    grab(axes: string[], inputType?: IInputType, event?: any): void;
-    restore(inputEvent?: any): void;
+    private createAnimationParam(pos, duration, option);
+    grab(axes: string[], option?: ChangeEventOption): void;
```

Comment:
`option` is really optional? There's no grab function call in which `option` is omitted

Reply:
Ok you're right. I fit it.